CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ project(EVT
         LANGUAGES CXX C
 )
 
-add_library(${PROJECT_NAME} STATIC)
+add_library(${PROJECT_NAME} STATIC src/dev/ADCMux.cpp include/EVT/dev/ADCMux.hpp samples/adc_mux/main.cpp)
```

Comment:
You shouldn't be adding the code as a library, you should be able to just add it to the `target_sources`